style.css

```
@@ -39,4 +39,54 @@ p {
 
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
+
+  padding: 20px 20px 5px 20px;
+}
+
+.title {
+  font-weight: bold;
+  font-size: 1.2em;
+}
+
+.content {
+  flex: 1;
+  overflow-y: auto;
+}
+
+.add {
+  padding: 10px;
+  display: flex;
+  border-top: 1px solid black;
+}
+
+input {
+  border: 1px solid black;
+  border-radius: 1rem;
+  flex-grow: 1;
+  margin-right: 10px;
+  padding: 5px;
+  text-align: center;
 }
+
+.add_button {
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  background-color: #2ec1ac;
+  text-align: center;
+  color: white;
+}
+
+.todo {
+  margin: 0 0 10px 0;
+  display: flex;
+  justify-content: center;
+}
+
+.todo > span {
+  flex-grow: 1;
+}
+
+.trashcan {
+  max-height: 1rem;
+}
```

```suggestion
.trashcan {
  max-height: 1rem;
  opacity: 0;
  transition: opacity ease-in .5s;
}

.title:hover .trashcan {
  opacity: 1;
}
```

css의 transition이라는 기능을 이용하면 특정 조건이 만족되었을때 엘리먼트의 css를 변경할수가 있어요!
이런식으로 todo item에 마우스를 올려놓았을때 (hover 했을 때) 휴지통 아이콘이 나타나도록 할수 있습니다!

---

오....저도 이건 지금알았어요,,! 같이 고칩시당 ㅎㅎㅎ